packages/core/src/components/MonthPicker/MonthPicker.jsx

```diff
@@ -131,11 +131,9 @@ export class MonthPicker extends React.PureComponent {
   }
 
   renderLabel() {
-    const classes = classNames('ds-u-font-weight--bold', this.props.labelClassName);
     return (
       <FormLabel
         className="ds-u-visibility--screen-reader"
-        labelClassName={classes}
```

Comment:
I removed `labelClassName` prop in `MonthPicker` because this `FormLabel` was hidden regardless. I'm not sure if anyone was actually using the prop or if there is a legitimate use case I'm unaware of. 
@line47